src/Vehicle/Vehicle.cc

```diff
@@ -495,12 +493,6 @@ void Vehicle::_mavlinkMessageReceived(LinkInterface* link, mavlink_message_t mes
                 emit messagesLostChanged();
         }
     }
-    // TODO: Set Signing AppSetting Based on First Message?
-    const bool mavlinkSigning = message.incompat_flags & MAVLINK_IFLAG_SIGNED;
-    if (mavlinkSigning != _mavlinkSigning) {
```

Comment:
I would also re-add this, as this is the only way to know if it is actually working or not.

Reply:
Yeah, that will be part of the remaining work on showing signing status somehow.